src/index.ts

```diff
@@ -0,0 +1,8 @@
+const sum = (a: number, b: number) => {
```

Comment:
```suggestion
export const sum = (a: number, b: number) => {
```
This module needs to export something or the tests will fail.